### ISSUE TITLE
Playback density rearrangement and reset

### ISSUE THREAD

**Issue by kdarras** [ecomontec/ecoSound-web#380]
The current listening mode button with the CD icon is not intuitive.
Since it is related to zooming, this option should be moved to the expandable options menu attached to the zoom button, with the same design as in #378 .
The option should be called "Zoom density" and the user should get a choice of densities measured in px/sec or FFT windows/s (this needs a decision regarding #258 )

**Comment by LiuDilongNJ** [comment]
Does this zoom in/out function use the centre of the current image as a reference? Does it need to zoom in and out in both horizontal and vertical coordinates?

**Comment by kdarras** [comment]
Yes, the "simple" zoom in and out functions would use the centre of the current view and zoom in and out by the percentage indicated by the user, in both time and frequency. Such a function was previously implemented.

**Comment by kdarras** [comment]
Generally works but the following adjustments are necessary:

- deactivate automatic playback after selecting "zoom into X px/s" and also the activation of continuous playback - the users should activate it themselves from now on
- replace "Please enter a positive number less than 100 for zoom in" by "Please enter a positive number smaller than 100 for zooming in" and add "by" so that the labels read "zoom in/out BY X%" (my mistake)
- if you think it still looks good, we can remove the "zoom" from each one of the four lines and have a "Zoom options" header instead to be less repetitive
